src/testdir/Make_all.mak

```diff
@@ -349,7 +349,8 @@ NEW_TESTS = \
 	test_zip_plugin \
 	test_alot_latin \
 	test_alot_utf8 \
-	test_alot
+	test_alot \
+	test_getcellpixels
```

Comment:
Please read the comment above:
```make
# Please keep sorted up to test_alot.
```
So this should come between `test_ga` and `test_getcwd`.